packages/jaeger-ui/src/components/TracePage/TraceSpanView/index.test.js

```diff
@@ -38,7 +38,7 @@ describe('<TraceSpanView>', () => {
   it('does not explode', () => {
     expect(wrapper).toBeDefined();
     expect(wrapper.find('.title--TraceSpanView').length).toBe(1);
-    expect(wrapper.find('.span-view-table').length).toBe(3);
+    expect(wrapper.find('.span-view-table').length).toBe(2);
```

Comment:
why is this count changing?

Reply:
Since the re-write of the Table in 4.x, there doesn't exist an extra `withStore(Table)` dom element with the class name `span-view-table`.